Makefile

```diff
@@ -152,7 +152,8 @@ help: ## Display this help.
 
 ##@ Development
 
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+# run make bundle instead
+manifests: controller-gen
```

Comment:
remove help to avoid my mistake of running manifests instead of bundle. Or there is scenarios that is good to run only manifests?

Reply:
If we did not have the nullables target we added, perhaps manifests would be more useful to run occasionally.. but we still wanna keep it around, please add back the comments around what it does.